create_virtualmachine_with_subnet_in_different_resource_group.yml

```diff
@@ -1,7 +1,16 @@
 # Description
 # ===========
 # This playbook create an Azure VM with public IP, and subnet in another resource group.
-
+# This Script will create two resource group, defined by variables resource_group and resource_group_secondary. 
+# Then it creates VM in resource_group and virtual network with defined subnets,nsg rules in resource_group_secondary.
+# To be able to run this script you will need to be in active session of azure subscription where you want to create this infrastruture. 
+# You have two options to achive this
```

Comment:
As we add the part in the readme, we can remove the following how to run part to avoid redundant